docs/streams/upgrade-guide.html

```diff
@@ -136,7 +136,12 @@ <h3 class="anchor-heading"><a id="streams_notable_changes" class="anchor-link"><
     <h3><a id="streams_api_changes_370" href="#streams_api_changes_370">Streams API changes in 3.7.0</a></h3>
     <p>
         IQv2 supports <code>RangeQuery</code> that allows to specify unbounded, bounded, or half-open key-ranges, which return data in ascending (byte[]-lexicographical) order (per partition).
-        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-985%3A+Add+reverseRange+and+reverseAll+query+over+kv-store+in+IQv2">KIP-985</a> extends this functionality by adding <code>.withDescendingKeys()<code> to allow user to receive data in descending order.
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-985%3A+Add+reverseRange+and+reverseAll+query+over+kv-store+in+IQv2">KIP-985</a> extends this functionality by adding <code>.withDescendingKeys()</code> to allow user to receive data in descending order.
+    </p>
+
+    <p>
+        KIP-992 add two new query types, namely <code>TimestampedKeyQuery</code> and <code>TimestampedRangeQuery</code>. Both should be used to query a timestamped key-value store, to retrieve a <code>ValueAndTimestamp</code> result. The existing <code>KeyQuery</code> and <code>RangeQuery</code> are changes to always return the value only, also for timestamped key-value stores.
```

Comment:
Make `KIP-992` here the link directly and remove it from the end of the paragraph

Reply:
ok